macosx/VDKQueue/VDKQueue.mm

```diff
@@ -21,8 +21,10 @@
 
 #import "VDKQueue.h"
 
-#import <unistd.h>
-#import <fcntl.h>
+#import <Foundation/Foundation.h>
```

Comment:
Please revert: that import is already present in "VDKQueue.h"

Reply:
We're just experimenting here. You can see that the build errors doesn't make sense, so not too sure what to do here.

Reply:
I'm looking at it since you called me for help (https://github.com/transmission/transmission/pull/7191#issuecomment-2712802738). :)

So far, all we know is that latest Xcode is building the code fine.
I don't have currently a setup ready with an older Xcode, but I'll see what I can do.

At first I thought of the various degrees of C++ support across Xcode releases:
https://developer.apple.com/xcode/cpp/#c++20
But then we know that code was building fine with C++17.